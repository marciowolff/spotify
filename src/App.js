import React from 'react';
import Routes from './pages/routes';
import { Provider } from 'react-redux';

import configStore from './reducers/config-store';
const store = configStore();

import './App.scss';

const App = () => (
  <Provider store={store}>
    <Routes />
  </Provider>
);

export default App;
