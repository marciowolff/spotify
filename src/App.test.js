import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';
it('should render App component', () => {
 const component = mount(<App />);
 expect(toJson(component)).toMatchSnapshot();
});