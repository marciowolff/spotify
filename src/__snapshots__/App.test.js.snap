// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App component 1`] = `
<App>
  <h1>
    Teste
  </h1>
</App>
`;
